hedera-mirror-test/src/test/java/com/hedera/mirror/test/e2e/acceptance/steps/ContractFeature.java

```diff
@@ -131,6 +132,11 @@ public void verifyContractFunctionCallMirror() {
         verifyContractExecutionResultsByTransactionId();
     }
 
+    @Then("the mirror node Web3 REST API should execute simulations")
+    public void verifyContractSimulations() {
```

Comment:
Again, not sure why using simulations terminology. Let's use "contract call" to be consistent with the API endpoint.
```suggestion
    @Then("I call the contract via the mirror node REST API")
    public void restContractCall() {
```